third_party/boringssl/src/crypto/x509/by_dir.c

```diff
@@ -131,7 +131,7 @@ static int dir_ctrl(X509_LOOKUP *ctx, int cmd, const char *argp, long argl,
   switch (cmd) {
     case X509_L_ADD_DIR:
       if (argl == X509_FILETYPE_DEFAULT) {
-#if defined(COBALT)
+#if defined(OPENSSL_SYS_STARBOARD)
```

Comment:
Are `STARBOARD` and `OPENSSL_SYS_STARBOARD` interchangeable under `third_party/boringssl/src`?
Do we have a scheme for which to use when?

Reply:
They are interchangeable in BoringSSL yes. the rest of the changes are gated with `OPENSSL_SYS_STARBOARD` in there, so this is just to make it more uniform.

I think in general, we should be using upstream package conventions everywhere, and these are different per package. In `//base`, it's `BUILDFLAG(IS_platform)` and we should migrate. In Skia it's a different pattern and so on.

The package build code in `gn` or `gni` files just needs to have a few lines of code that maps our build ( `is_starboard` or such ) to platform macro.